cli/admin/README.md

```diff
@@ -86,6 +86,7 @@ From that point onwards, you can check the available commands by typing `help`:
 > help
 All available commands:
         bk cleanup : Removes orphan BookKeeper Ledgers that are not used by any BookKeeperLog.
+        bk delete-ledgers <log-id> <ledger-id>: Deletes all the ledgers for a bookkeeper log starting with and including given ledger-id as the starting index.
```

Comment:
It is not clear the reason why an user should use this feature and the consequences

Reply:
Right, maybe we could add something similar to what is mentioned in the header of this file: `Note that this is a destructive operation that should be used only to remove missing or corrupted ledgers that prevent a container from recovering.`

Reply:
Updated.